tests/mail/tests.py

```diff
@@ -1469,12 +1469,25 @@ class SMTPBackendStoppedServerTest(SMTPBackendTestsBase):
     ("RuntimeError: threads can only be started once").
     """
 
-    def test_server_stopped(self):
+    def setUp(self):
+        super(SMTPBackendStoppedServerTest, self).setUp()
+        self.backend = smtp.EmailBackend(username='', password='')
+
+    def test_1_server_stopped(self):
```

Comment:
It would be better to rename this and make the `setUp` refactor in a separate commit. No need to include a number in the test method name.